### DISCUSSION TITLE
Turn SD-card symbol into Pause during printing?

### DISCUSSION THREAD

**Discussion by DenPrint** [moonglow/FlashForge_Marlin#54] [Q&A]
Hello,
Im just a beginner with marlin, so maybe im asking something thats not possible.

While printing in the SD card button greyout (and that makes sense).
![homescreen default](https://user-images.githubusercontent.com/86926224/130057916-c8a1a69d-8d65-4f36-ac88-d2c944f257eb.jpg)

But is it possible to make the SD-card symbol turn into a pause button while printing?
![pause](https://user-images.githubusercontent.com/86926224/130057982-a42bb7ee-677c-45aa-8ad8-cb2a78eb8ac7.jpg)

With a TFT screen that would make it easier to pause/stop printing then first go into the configuration screen.

Thanks

**Comment by OneMonkeyArmy** [comment]
Possible to achieve, you can change graphics for any given situation, like pause or card is removed... etc  But you have to do it in code and add needed graphics to the firmware. Nothing that can be done easily in any config file...

https://user-images.githubusercontent.com/84219749/139527592-137d8332-9369-4107-a575-6021b332b186.mp4
